soh/src/overlays/actors/ovl_Door_Ana/z_door_ana.h

```diff
@@ -11,6 +11,8 @@ typedef void (*DoorAnaActionFunc)(struct DoorAna*, PlayState*);
 typedef struct DoorAna {
     /* 0x0000 */ Actor actor;
     /* 0x014C */ ColliderCylinder collider;
+                 s32 useUselessCylinder;
+                 ColliderCylinder uselessCollider;
```

Comment:
I think either a comment saying this is for the collision viewer or making the variable name clear its for the collision viewer (and ours) would be good (or both!)

Reply:
same applies to all cases of these.